app/views/users/_dashboard_deck.html.erb

```diff
@@ -1,11 +1,11 @@
 <div class="block">
   <div class="block-image">
-    <%= link_to "/learn/#{@user.id}/#{deck.id}" do %>
+    <%= link_to "/users/#{@user.id}/#{deck.id}" do %>
       <%=image_tag image_path("temp/dashboard-img-#{rand(1..4)}.jpg") %>
     <% end %>
   </div>
   <div class="block-details">
-    <%= link_to "/learn/#{@user.id}/#{deck.id}", class: 'title-link' do %>
+    <%= link_to "/users/#{@user.id}/#{deck.id}", class: 'title-link' do %>
```

Comment:
i don't see a url_helper for this or the rest of the helper comments below.
is it possible/best practice to create helpers for non-resourceful routes?
having some difficulty finding clear documentation on this ...